src/wp-admin/security.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * Security Administration Screen.
+ *
+ * @package ClassicPress
+ * @subpackage Administration
+ */
+
+/** ClassicPress Administration Bootstrap */
+require_once( dirname( __FILE__ ) . '/admin.php' );
+
+$title = __('Security');
+
+require_once( ABSPATH . 'wp-admin/admin-header.php' );
+
+$tabs = [
+    'users' => [
+        'title' => __('For Users'),
+        'classes' => ['nav-tab']
+    ],
+    'developers' => [
+        'title' => __('For Developers'),
+        'classes' => ['nav-tab']
+    ]
+];
+$active_tab = (in_array(@$_GET['tab'], array_keys($tabs)))
+    ? $_GET['tab']
+    : 'users';
+$tabs[$active_tab]['classes'][] = 'nav-tab-active';
+
+?>
+<div class="wrap">
+  <h1><?php echo esc_html( $title ); ?></h1>
+  <h2 class="nav-tab-wrapper">
+<?php   foreach ($tabs as $tab => $info): ?>
+    <a href="/wp-admin/security.php?tab=<?=$tab?>" class="<?=join(' ', $info['classes'])?>"><?=$info['title']?></a>
+<?php   endforeach; ?>
+  </h2>
+<?php
+switch ($active_tab):
+    case 'users':
+?>
+
+  <div class="card">
+    <h2><?php _e( '"Security First"' ); ?></h2>
+    <p><?php _e( "Security is important to business and it’s important to us here at ClassicPress. By bringing security forward to a place of greater prominence within the admin interface, we create a more streamlined experience for both users and developers." ); ?></p>
+    <p><?php _e( "As ClassicPress continues to evolve, the Security page will become the hub for all security features for ClassicPress core and 3<sup>rd</sup> party plugins that choose to support it." ); ?></p>
+    <p><?php printf(
+        /* translators: canonical link to security forum */
+        __( "Watch this page and the ClassicPress <a href=\"%s\" target=\"_blank\">Security forum</a> for more news as development continues." ),
+        'https://forums.classicpress.net/security'
```

Comment:
Should we use `link.classicpress.net` here too? (I think probably yes for this one, because this is not a standard Discourse URL pattern)

Reply:
`/support/security`?

Reply:
This isn't really a support forum, how about `/forum/security`?

Just nitpicking at this point though, I have enabled both URLs for now (https://link.classicpress.net/support/security and https://link.classicpress.net/forum/security).